### ISSUE TITLE
SystemAlarms: ConfigError: Stabilization is hard to track down

### ISSUE THREAD

**Issue by solidgoldbomb** [TauLabs/TauLabs#1033]
When configuring a new multirotor, the defaults for the FlightMode settings includes "Manual" as one of the options.  This triggers a ConfigError.

It's a bit non-obvious what the source of the error is (ie. FlightMode list includes Manual).

This issue is intended to collect suggestions on how this particular config error could be better represented in the GCS.

**Comment by Kawobei** [comment]
I know this is old but.... I had same config error not setting ins when trying to use navigation modes. Some kind of indication where the problem would be nice.

**Comment by kubark42** [comment]
@Kawobei, yeah, our alarms don't have as much granularity as they could. We could consider adding an error field, like we have with several other specific alarms. This has the nasty downside of growing the UAVO but alarms that don't transmit enough information to easily resolve them are not ideal.
